first_hour_on_vwb/creating_a_data_collection.ipynb

```diff
@@ -0,0 +1,735 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Create a new workspace for the data collection\n",
+    "<a id=\"create-new-ws\"></a>\n",
+    "\n",
+    "In order to create a data collection, you must first create a new workspace. \n",
+    "Run the cell below to create a widget, then populate the widget's input fields and click the button to create your new workspace.\n",
+    "\n",
+    "Widget input parameters include:\n",
+    "- `Workspace Name`: Must be a string. This value is displayed in the Data Collection modal once the workspace is converted to a data collection, so the value should communicate the intended purpose (e.g. `<STUDY_NAME> Data Collection`).<br> While the Workbench UI and this widget require a workspace name to be provided, the CLI does not; if no workspace name is provided to the CLI, a UUID is generated instead. \n",
+    "- `Description`: Must be a string. This description will stay with the workspace after it becomes a data collection. Before you convert a workspace to a data collection, you can update this value in the UI.\n",
+    "- `Workspace ID`: Must be unique and consist only of lowercase letters, numbers and underscores. Provide a workspace ID that suggests something about the contents of the data collection you'd like to create and include the date of its creation, such as `<STUDY_NAME>_<YYMMDD>_dc_ws`. *You cannot change the workspace ID after workspace creation.* \n",
+    "- `Version`: Must be unique from subsequent versions (e.g., you cannot have two versions of a given data collection, both named \"1.0\"). Represents the first version of the data collection.\n",
+    "\n",
+    "\n",
+    "The output should resemble:\n",
+    "\n",
+    "```\n",
+    "Workspace successfully created.\n",
+    "ID:                <WORKSPACE_ID>\n",
+    "Name:              <WORKSPACE_NAME>\n",
+    "Description:       <DESCRIPTION>\n",
+    "Cloud Platform:    GCP\n",
+    "Google project:    <GOOGLE_PROJECT_ID>\n",
+    "Cloud console:     https://console.cloud.google.com/home/dashboard?project=<GOOGLE_PROJECT_ID>\n",
+    "Properties:\n",
+    "  terra-type: workspace\n",
+    "Created:           YYYY-MM-DD\n",
+    "Last updated:      YYYY-MM-DD\n",
+    "# Resources:       0\n",
+    "Successfully converted <WORKSPACE_ID> to data collection.\n",
+    "Created initial version of data collection: <VERSION>\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "class CreateWorkspaceWidget(object):\n",
```

Comment:
I noticed that with the 'create workspace' widget, the description field content, e.g. as shown here:
<img width="791" alt="CleanShot 2023-09-27 at 11 04 07@2x" src="https://github.com/DataBiosphere/terra-axon-examples/assets/115093/e8a77ed1-337f-4994-a97b-46da9867aa7b">
ends up with quotes around it in the created workspace description:
<img width="551" alt="CleanShot 2023-09-27 at 11 04 21@2x" src="https://github.com/DataBiosphere/terra-axon-examples/assets/115093/09394f3f-f2e4-4ed3-bbf0-73fb9a3d3a9c">


The user can go in to the UI and edit that, but it would be nice for them to not have to do that.

Reply:
I agree that having to manually remove delimiters via the UI  is annoying; however, when I tested with single quotes, the single quotes still appeared (with CLI v0.411.0). I have therefore changed the widget such that all leading and trailing whitespace is stripped but the string is undelimited, which does not add delimiters to the description field.